src/main/java/bibliotecame/back/Book/BookRepository.java

```diff
@@ -19,20 +19,20 @@
     Optional<BookModel> findByTitleAndAuthorAndPublisherAndYear(String title, String author, String publisher, int year);
 
     @Query(value = "select b from BookModel b where" +
-            " b.title like %:toCompare%" +
-            " or b.author like %:toCompare%" +
-            " or b.publisher like %:toCompare%" +
+            " b.title like %:search%" +
```

Comment:
The comparison should be case insensitive.

Look at this [article](https://stackoverflow.com/questions/37178520/jpql-like-case-insensitive/41988897)

Reply:
done